### DISCUSSION TITLE
It's Brand

### DISCUSSION THREAD

**Discussion by MR-ZIHAD** [community/community#18902] [Other Feature Feedback, Questions, & Ideas]
Occupation | I usually help social media users
-- | --
Github Developer, Ethical hacking & Programming.
Bangladesh privet cyber security agency

Zihad Hossain Rafi is a Bangladeshi Github Developer. Zihad Hossain Rafi helps ordinary social media users. He informs them about the security of their accounts. He started his Journey since 2018. Zihad Hossain Rafi became popular on Facebook in a short time. He brought back almost all the hacked social media accounts including Facebook and Twitter.And whose account he returned.

**Comment by justin-2028** [comment]
I'm not sure how to help you here, but I'm guessing its a self-promotion. Even so, I wish you the best for your career, have a good day 👍
